inttest/capi-docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM kindest/node:v1.27.1
```

Comment:
Looks like this was added accidentally.  Also it doesn't finish with a \n. Other than that LGTM.

Reply:
oh yeah, `git add`ing random stuff strikes again 😄

Reply:
removed